section2/exercises/comparisons.js

```diff
@@ -49,46 +49,50 @@ you develop good habits that follow best practice, from now on, use the strict c
 
 // YOU DO: log the result of the comparison: is numberTeachers not equal to numberStudents?
 // this should log: true
-
+console.log('Is numberTeachers not equal to numberStudents?', numberTeachers !== numberStudents);
```

Comment:
✅

Reply:
Challenge yourself on your own time: What is the difference between `!=` and `!==`?

Reply:
Got it! Just like === means strictly equals and checks value and type, the same can be said for !==, but not equal. Similarly, the == operator checks the value only, not the type, and the same is the case for !=, but not equal to.